pravega/bookkeeper-operator
# Issue 1: Initial code changes to create bookkeeper-operator

Signed-off-by: pbelgundi <prajakta.belgundi@emc.com>

### Change log description
Code for Bookkeeper Operator to manage the pravega-bookkeeper cluster.

### Purpose of the change
Fixes #1

### How to verify it
Should be able to build a pravega/bookkeeper-operator image and deploy the same.
Should  be able to create a new Bookkeeper cluster and manage the same using this operator.
